src/atoms/TextInput/HelpText.native.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Text from '../Text';
+import Space from '../Space';
+import Flex from '../Flex';
+import View from '../View';
+
+const styles = {
+  color({ disabled }) {
+    if (disabled) {
+      return 'shade.930';
+    } else {
+      return 'shade.950';
+    }
+  },
+};
+
+const HelpText = ({ children, disabled }) => {
```

Comment:
Can we have `<ErrorText />` as part of this file? Something like this
`<HelpText color={styles.helpText.color} disabled={disabled}>{errorText | helpText}</HelpText>`. 
Both `<ErrorText />` and `<HelpText />` have similar props.

Reply:
I kept them separate in case they need to diverge at any point. Easier to iterate

Reply:
They won't diverge. Let's keep them in one place. Right now it feels too granular which is not needed

Reply:
We can rename the file as `Text.native.js`